examples/NDSL/02_NDSL_basics.ipynb

```diff
@@ -125,12 +124,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "## Change this to Quantity\n",
-    "\n",
```

Comment:
The code below is using `Quantity` objects. Seems like a lost todo. Or is there more refactoring needed here?